opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -38,7 +38,11 @@ def test_resource(self):
     def test_resource_empty(self):
```

Comment:
wonder if this test should be renamed since it's not using the empty resource (there's a separate `create_empty` method for empty resources)